hbase-server/src/main/java/org/apache/hadoop/hbase/io/FileLink.java

```diff
@@ -307,9 +307,9 @@ private FSDataInputStream tryOpen() throws IOException {
           assert(in.getPos() == pos) : "Link unable to seek to the right position=" + pos;
           if (LOG.isTraceEnabled()) {
             if (currentPath == null) {
-              LOG.debug("link open path=" + path);
+              LOG.trace("Link open path={}", path);
```

Comment:
Presuming the change in log level intentional?

Reply:
looking at the history of this code (37ab60bcc577f669015f5b4b3045f4207795ad34 and feede7abd8d appear to be the relevant commits) it's not clear why this was at debug in the first place. my guess is so we'd get a DEBUG message per series of `tryOpen` calls without having to see every attempt while traversing the possibilities.

I think it's still valuable to have a debug message that says "hey we're trying to resolve a FileLink and we're currently just starting to look at locations" but we should do that directly with a debug message before the for loop.